tests/midlets/background/PauseMIDlet.java

```diff
@@ -0,0 +1,32 @@
+package tests.background;
```

Comment:
I'm trying to make all the tests I write under tests/midlets/ to be usable both manually and in automated tests, but there are probably some that only work in automated tests.